rpm/amber-mpris-qt6.spec

```diff
@@ -0,0 +1,76 @@
+Name:       amber-mpris-qt6
+
+Summary:    Qt and QML MPRIS interface and adaptor
+Version:    1.1.0
+Release:    1
+License:    LGPLv2+
+URL:        https://github.com/sailfishos/amber-mpris
+Source0:    %{name}-%{version}.tar.bz2
+Requires(post): /sbin/ldconfig
+Requires(postun): /sbin/ldconfig
+BuildRequires:  pkgconfig(Qt6Core)
+BuildRequires:  pkgconfig(Qt6DBus)
+BuildRequires:  pkgconfig(Qt6Qml)
+
+%description
+%{summary}.
+
+%package devel
+Summary:    Development files for %{name}
+Requires:   %{name} = %{version}-%{release}
+Conflicts:  amber-mrpis-devel
+
+%description devel
+Development files for %{name}.
+
+%package -n amber-qml-plugin-mpris-qt6
+Summary:    QML plugin for %{name}
+Requires:   %{name} = %{version}-%{release}
+
+%description -n amber-qml-plugin-mpris-qt6
+QML plugin for %{name}.
+
+%prep
+%setup -q -n %{name}-%{version}
+
+%build
+
+export PATH=$PATH:%{_qt6_bindir}
+%qmake_qt6 VERSION=`echo %{version} | sed 's/+.*//'`
+%make_build
+
+%install
+rm -rf %{buildroot}
```

Comment:
Btw there are these details to clean up in the original .spec already. Can do on both already and avoid some later change. Namely:
- this rm can be removed
- all %defattr lines in %files sections can be removed